app/views/team-members/team-member-permissions.njk

```diff
@@ -30,62 +30,154 @@ Edit team member permissions - GOV.UK Pay
   </dl>
   <form id="role-update-form" method="post" action="{{editPermissionsLink}}">
     <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
-    {{
-      govukRadios({
-        idPrefix: 'role-input',
-        name: 'role-input',
-        fieldset: {
-          legend: {
-            text: 'Permission level',
-            classes: 'govuk-fieldset__legend--s'
-          }
-        },
-        items: [
-          {
-              value: admin.id,
-              id: "role-admin-input",
-              text: 'Administrator',
-              classes: 'govuk-label--s',
-              checked: admin.checked,
-              hint: {
-                html: '<ul class="govuk-list pay-text-grey">
-                        <li>View transactions</li>
-                        <li>Refund payments</li>
-                        <li>Manage settings</li>
-                      </ul>'
-              }
+    {% if serviceHasAgentInitiatedMotoEnabled %}
```

Comment:
I think the duplication here can be avoided, as the entries seem to have exactly the same content it feels prime for getting out of sync with each other.

It might work something like

```njk
{% set roleOptionItems = [{
    value: admin.id,	
    id: "role-admin-input"
  }, {
    // ...
  }] 
%}
 
{% if serviceHasAgentInitiatedMotoEnabled %}

{% set roleOptionItems = roleOptionItems.push({
      value: viewAndInitiateMoto.id,
      id: "role-view-and-initiate-moto-input"
    }, {
       value: viewRefundAndInitiateMoto.id,
       id: "role-view-refund-and-initiate-moto-input"
    }) 
%}

{% endif %}

{% govukRadios({
  items: roleOptionItems
}) %}
```

This solution hasn't actually been tried, the syntax may be whack.

Reply:
They’re not quite the same: they have slightly different hints. But I’ll see what I can do.

Reply:
Ah I didn't notice the extra hint implicit with the flag being set. This doesn't need to block the pr

Reply:
I think I will leave it for now. While adding extra roles to the array is fairly straightforward, also going back and modifying all the hints would be a bit gnarly!